### PR TITLE
Add QOS parameter

### PR DESCRIPTION
**Pull request by sgbaird**
Add QOS parameter per #1610

### CODE REVIEW

**Comment by sgbaird** [comment]
check_format seems to subtract and add the same lines (i.e. the changes are identical). Odd..

**Comment by gwenzek** [comment]
there is a actually is a one space diff. You can run `make format` to apply them.

**Review by gwenzek — APPROVED** [review]

File: submitit/slurm/slurm.py [hunk 1]
```diff
@@ -409,6 +409,7 @@ def _make_sbatch_string(
     array_parallelism: int = 256,
     wckey: str = "submitit",
     map_count: tp.Optional[int] = None,  # used internally
+    qos: tp.Optional[str] = None, # quality of service
```
Comment (gwenzek) [inline submitit/slurm/slurm.py:412]: Could you move this one line up , above "map_count" ?

**Review by gwenzek — APPROVED** [review]

File: submitit/slurm/slurm.py (hunk 1 above)
Comment (sgbaird) [reply submitit/slurm/slurm.py:411]:
```suggestion
    qos: tp.Optional[str] = None, # quality of service
```

**Review by sgbaird — COMMENTED** [review]
I think that should do it

**Comment by gwenzek** [comment]
Closed with  a559304fce4a0461a8352281f4a24a55dcfe4648

**Pull request closed without merging**